__tests__/app.test.js

```diff
@@ -56,3 +56,47 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe('"/api/reviews/:reviews_id', () => {
+  describe("GET", () => {
+    test("200: returns an object containing information on requested review (with comment count)", () => {
+      return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual({
+            review_id: 3,
+            title: "Ultimate Werewolf",
+            designer: "Akihisa Okui",
+            owner: "bainesface",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "We couldn't find the werewolf!",
+            category: "social deduction",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+            comment_count: "3",
+          });

```

Comment:
Using toEqual is a brittle test for this endpoint because it doesn't leave room for extension. I'd like you to amend the testing here to be more flexible. This means using 'expect.objectContaining' or '.toMatchObject' instead